tests/test_hec2d.cpp

```diff
@@ -392,8 +392,8 @@ TEST( MeshHec2dTest, model_505 )
   count = MDAL_D_valueCount( ds );
   ASSERT_EQ( 73, count );
 
-  value = getValue( ds, 10 );
-  EXPECT_DOUBLE_EQ( 33.000003814697266, value );
+  value = getValue( ds, 50 );
+  EXPECT_TRUE( MDAL::equals( 34.785, value, 0.001 ) );
```

Comment:
why the value changed?

Reply:
with dry/wet fixed, getValue( ds, 10 ) = NaN, 
but better to keep the test on NaN and add a test to not NaN